README.md

```diff
@@ -141,19 +141,20 @@ cat record.json | bundle exec traject -c lib/traject/config/folio_config.rb -s r
 ```
 note that this approach doesn't use the `FolioClient` to make API calls, so the burden is on the user to create a fully-formed `FolioRecord` prior to indexing.
 
-## environments
-### Symphony ILS (Sirsi)
-MARC binary data is dumped into files (each containing ~500k records) on the Symphony servers. These dumps happen hourly (containing every changed MARC record during that calendar day), nightly (containing every record changed the previous day), and monthly (containing every exportable MARC record in Symphony). Hourly and nightly dumps also include a `del` file, containing a catkey-per-line of records that have been deleted or retracted.
+It's also possible to index a group of druids, mimicking the process from SDR.
+`SOLR_URL=http://localhost:8983/solr/blacklight-core bundle exec traject -i xml -c lib/traject/config/sdr_config.rb druidslist.txt`
+You can create a `druids.txt` file containing a list of newline delimitd druids.
 
-Additional data comes from a course reserves data dump, also on the Symphony servers. Course reserves files are pipe `|` separated values (PSV) files which are read in during the indexing process and used to enhance MARC records during the transform process.
+### Data sources 
+Our catalog data comes from queries to the FOLIO Postgres database.
```

Comment:
And from SDR. Maybe this statement should be under the FOLIO heading?

Reply:
Thanks @jcoyne. Updated.